### DISCUSSION TITLE
Thumbnails not rendering in collection table

### DISCUSSION THREAD

**Discussion by danielbuechele** [directus/directus#17494] [Archive]
I have a collection where each item has an image assigned. Opening an individual item from the collection, I can see the image, change it, etc.

However, in table view, the thumbnails are not rendering. The URL requested for the image is `/assets/undefined?key=system-small-cover&download=true`, resulting in HTTP403 errors. To me it looks like the image ID is not passed correctly to the component, causing this image not to render, and instead show a fallback image:
<img width="291" alt="image" src="https://user-images.githubusercontent.com/685740/218290580-0a239e11-3923-4a4d-aca3-a3295da7baa0.png">

Switching to cards layout shows the images as expected. I'm using Google Cloud Storage to host my images. Is this a bug/known issue, or a misconfiguration on my side?

**Comment by rijkvanzanten** [comment]
Feels like a bug! The `/assets/undefined` is very suspicious. Mind opening [a new Bug Report](https://github.com/directus/directus/issues/new?assignees=&labels=&template=bug_report.yml)? Thanks!

**Reply by danielbuechele** [reply]
Created #17541
